src/register/signals.md

```diff
@@ -335,9 +335,10 @@ This still allows you to use `#[signal]` and will register each signal declared
 The typed signal API is generated for your class under the following conditions:
 
 - Your class declares a `Base<T>` field.
-- You have a `#[godot_api]` block (empty if necessary).
-   - This is a technical limitation that may be lifted in the future.
 - You do not opt out from typed signals with `no_typed_signals`.
+- You have a `#[godot_api]` block  on your class `impl` (empty if necessary).
+   - Requires seperate blocks for both `impl Monster` and `impl INode3D for Monster`
+   - This is a technical limitation that may be lifted in the future.
```

Comment:
It only requires the attribute on the inherent `impl` block, not the trait one -- or am I missing something?

If yes, you can combine this to 1 point:

```suggestion
- You have a `#[godot_api]` block  on your class' inherent `impl` block (empty if necessary; not the `I*` trait `impl`).
  This is a technical limitation that may be lifted in the future.
```